setup.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Nokia
+# Licensed under the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+from setuptools import Command, setup
+
+
+THISDIR = os.path.join(os.path.dirname(os.path.abspath(__file__)))

```

Comment:
Elsewhere has been used CURRENT_DIR for the same purpose. Maybe either THISDIR or CURRENT_DIR everywhere.